src/Unity.Mathematics/math.cs

```diff
@@ -6704,6 +6704,18 @@ public static float4 chgsign(float4 x, float4 y)
             return asfloat(asuint(x) ^ (asuint(y) & 0x80000000));
         }
 
+        /// <summary>
+        /// Read 32 bits of data in little endian format.
+        /// </summary>
+        /// <param name="pBuffer">Memory address to read from.</param>
+        /// <returns>32 bits in little endian format.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static unsafe uint read32_little_endian(void* pBuffer)
```

Comment:
Once available on all Unity versions where this is needed, we should probably use [Unsafe.ReadUnaligned](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.unsafe.readunaligned?view=net-7.0#system-runtime-compilerservices-unsafe-readunaligned-1(system-void*))